isis/src/lro/objs/LroWideAngleCamera/LroWideAngleCamera.cpp

```diff
@@ -291,8 +292,18 @@ namespace Isis {
     }
 
     if (nvals <= 0) {
-      QString mess = "Kernel pool keyword " + key + " not found!";
-      throw IException(IException::Programmer, mess, _FILEINFO_);
+      try{
+        PvlObject NaifKeywords = getStoredNaifKeywords();
+        PvlKeyword kw = NaifKeywords[key];
+        IntParameterList parms;
+        for (int i = 0; i<kw.size(); i++){
+          parms.push_back(toInt(kw[i]));
+        }
+        return parms;
+      } catch(...){
+        QString mess = "Kernel pool keyword " + key + " not found!";
+        throw IException(IException::Programmer, mess, _FILEINFO_);
+      }
```

Comment:
This shouldn't be in a try catch and catching ANY exception. Instead this should be explicitly checking that the keyword exists in the Naif Keywords group and then directly raising an exception if it doesn't.